lib/_http_common.js

```diff
@@ -195,3 +195,11 @@ function httpSocketSetup(socket) {
   socket.on('drain', ondrain);
 }
 exports.httpSocketSetup = httpSocketSetup;
+
+/** Verifies that the given val is a valid HTTP token
```

Comment:
It doesn't mean that we cannot use it. And it is [already](https://github.com/nodejs/node/blob/master/lib/internal/child_process.js#L363-L383) [used](https://github.com/nodejs/node/blob/master/lib/timers.js#L168-L170).